Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.h

```diff
@@ -91,6 +90,28 @@ class RemoteLayerTreeDrawingAreaProxy : public DrawingAreaProxy {
 
     bool shouldCoalesceVisualEditorStateUpdates() const override { return true; }
 
+    // displayDidRefresh is sent to the WebProcess, and it responds
+    // with a commitLayerTree message (ideally before the next
+    // displayDidRefresh, otherwise we mark it as missed and send
+    // it when commitLayerTree does arrive).
+    enum CommitLayerTreeMessageState { CommitLayerTreePending, NeedsDisplayDidRefresh, MissedCommit, Idle };
+    struct ProcessState {
+        CommitLayerTreeMessageState commitLayerTreeMessageState { Idle };
+        TransactionID lastLayerTreeTransactionID;
+        TransactionID pendingLayerTreeTransactionID;
+
+#if ENABLE(THREADED_ANIMATION_RESOLUTION)
+        Seconds m_acceleratedTimelineTimeOrigin;
```

Comment:
This is a struct.  These should probably not have the m_ prefix like the ones above don't have.